src/main/resources/elasticsearch.template.properties

```diff
@@ -0,0 +1,36 @@
+# ============ elasticsearch configuration for picturesafe-search =============
+#
+# picturesafe-search requires a configuration file named 'elasticsearch.properties' in the classpath,
+# in which the following default parameters can be overwritten.
+
+
+# --------------------------- Client configuration ----------------------------
+
+## List of host addresses to connect to (blank separated, format <hostname>:<port> or <ip>:<port>)
+#elasticsearch.hosts=localhost:9200
+
+## automatically discover nodes from a running Elasticsearch cluster and set them to an existing RestClient instance
+#elasticsearch.sniffer.enabled=false
+
+# --------------------------- Index configuration -----------------------------
+
+#elasticsearch.index.alias=default
+
+## Index name prefix (property 'elasticsearch.index.alias' will be used when empty)
+#elasticsearch.index.name_prefix=
+
+## Index name date format ('yyyyMMdd-HHmmss-SSS' will be used when empty)
+#elasticsearch.index.name_date_format=yyyyMMdd-HHmmss-SSS
+
+#elasticsearch.index.number_of_shards=1
+
+#elasticsearch.index.number_of_replicas=0
```

Comment:
No comment?

Reply:
Speaks for itself, I think. Maybe refer to the Elastic documentation?